lib/airbrake-ruby/backtrace.rb

```diff
@@ -97,29 +97,18 @@ module Patterns
     end
 
     ##
+    # @return [Integer] how many first frames should include code hunks
+    CODE_FRAME_LIMIT = 10
```

Comment:
Ideally this should be auto-tunable option in truncator, e.g.
- try limit 64, check notice size
- try limit 32, check notice size
- etc